llvm/test/tools/llvm-objcopy/ELF/gap-fill.test

```diff
@@ -0,0 +1,149 @@
+# RUN: yaml2obj %s > %t
+
+## Verify section headers before we perform several testings.
```

Comment:
"tests" rather than "testings"

What is this actually supposed to be verifying though? You're using yaml2obj to create your input, so you should have full control over the output. You also aren't using this as a comparison to the later checks in the file (since those aren't checking section headers).

Reply:
I cleaned up this and other comments in this file.